openrouteservice/src/main/java/org/heigit/ors/routing/RoutingRequest.java

```diff
@@ -211,6 +212,10 @@ public void setIncludeCountryInfo(boolean includeCountryInfo) {
 		this.includeCountryInfo = includeCountryInfo;
 	}
 
+	public void setMaximumSpeed(double maximumSpeed){this.maximumSpeed = maximumSpeed;}
```

Comment:
Could you improve code formatting of the one-line methods and split it across lines such as:
```
public void setMaximumSpeed(double maximumSpeed) {
    this.maximumSpeed = maximumSpeed;
}

Reply:
Since a lot of other functions had one-line methods I followed this format. But I can change it.

Reply:
Yes, please.